R/create_eq.R

```diff
@@ -50,6 +50,24 @@ create_eq.polr <- function(lhs, rhs, ital_vars, use_coefs, coef_digits,
   list(lhs = lhs, rhs = rhs_final)
 }
 
+create_eq.clm <- function(lhs, rhs, ital_vars, use_coefs, coef_digits,
+                          fix_signs, model, ...) {
+  rhs$final_terms <- create_term(rhs, ital_vars)
+
+  if (use_coefs) {
+    rhs$final_terms <- add_coefs(rhs, rhs$final_terms, coef_digits)
+  } else {
+    rhs$final_terms <- add_greek(rhs, rhs$final_terms)
+  }
+
+  splt <- split(rhs, rhs$coef.type)
+  rhs_final <- lapply(splt$intercept$final_terms, function(x) {
+    c(x, splt$location$final_terms, "\\epsilon")
+  })
```

Comment:
Outside of minor changes here this function is almost a carbon copy of `create_eq.polr`. It's super minor any maybe not worth bothering, but if we could modify `create_eq.polr` so it maybe uses indexing instead of names where this has `intercept` and `location`, I think we could get away with `create_eq.clm` just being something like

```r
create_eq.clm <- create_eq.clm <- function(lhs, rhs, ital_vars, use_coefs, coef_digits,
                          fix_signs, model, ...) {
  create_eq.polr(lhs, rhs, ital_vars, use_coefs, coef_digits,
                          fix_signs, model, ...)
}
```